File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/youtube/playerResponse/Text.java
package com.github.kotvertolet.youtubejextractor.models.youtube.playerResponse;

import java.util.List;

public class Text {
    private List<RunsItem> runs;

    public List<RunsItem> getRuns() {
        return runs;
    }

    public void setRuns(List<RunsItem> runs) {
        this.runs = runs;
    }

    @Override
    public String toString() {
        return
                "Text{" +
                        "runs = '" + runs + '\'' +
                        "}";
    }
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/Text.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import com.google.gson.annotations.SerializedName

class Text {
    @SerializedName("runs")
    var runs: List<RunsItem>? = null
    override fun toString(): String {
        return "Text{" +
                "runs = '" + runs + '\'' +
                "}"
    }
}